modules/nw-exposing-service.adoc

```diff
@@ -58,7 +63,34 @@ Welcome to the MariaDB monitor.  Commands end with ; or \g.
 
 MySQL [(none)]>
 ----
+endif::nodeport[]
+ifdef::nodeport[]
+. To expose a node port for for the application, enter the following command. {product-title} automatically selects an available port in the `30000-32767` range.
```

Comment:
s/for for/for/
Also, "node port" is only used in this section, whereas **NodePort** is used throughout the first section ("Using a NodePort"). Pointing it out in case you want to align the doc more towards one format or the other.